app/controllers/idv/image_uploads_controller.rb

```diff
@@ -24,7 +24,13 @@ def image_upload_form
         analytics: analytics,
         uuid_prefix: current_sp&.app_id,
         irs_attempts_api_tracker: irs_attempts_api_tracker,
+        store_encrypted_images: store_encrypted_images?,
       )
     end
+
+    def store_encrypted_images?
```

Comment:
There is not great coverage here. The way I'd like to test this is to look for the reference UUIDs that get sent to the fake attempts API and make sure a file was written there. Since these aren't getting bubbled up to the attempts API yet we'll need to either come up with something else temporarily or hold off.